<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrinterBehavior</class>
 <widget class="QWidget" name="PrinterBehavior">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>484</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="4" column="0">
    <widget class="QGroupBox" name="BanneryGB">
     <property name="title">
      <string>Banners</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Starting Banner:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>startingBannerCB</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KComboBox" name="startingBannerCB">
        <property name="AttributeName" stdset="0">
         <string notr="true">job-sheets-default</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Ending Banner:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>endingBannerCB</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KComboBox" name="endingBannerCB">
        <property name="AttributeName" stdset="0">
         <string notr="true">job-sheets-default</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="PoliciesGB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Policies</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Operation Policy:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>operationPolicyCB</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Error Policy:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>errorPolicyCB</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KComboBox" name="errorPolicyCB">
        <property name="AttributeName" stdset="0">
         <string notr="true">printer-error-policy</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KComboBox" name="operationPolicyCB">
        <property name="AttributeName" stdset="0">
         <string notr="true">printer-op-policy</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QGroupBox" name="AllowGB">
     <property name="title">
      <string>Allowed Users</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="1" column="0">
       <widget class="QRadioButton" name="preventRB">
        <property name="text">
         <string>Prevent these users from printing </string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="KEditListBox" name="usersELB">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="buttons">
         <set>KEditListBox::Add|KEditListBox::Remove</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="allowRB">
        <property name="text">
         <string>Allow these users to print</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KEditListBox</class>
   <extends>QGroupBox</extends>
   <header>keditlistbox.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
